RELEASES.md

```diff
@@ -1,3 +1,159 @@
+Version 1.55.0 (2021-09-09)
+============================
+
+Language
+--------
+- [You can create bindings inside of a `@` bindings pattern.][85305] E.g. the
+  following is now possible.
+  ```rust
+  let outer_option = Some(42);
+
+  match outer_option {
+      option @ Some(ref value) => {
+         // We can now use both `option` and `value`.
+         ...
+      }
+      _ => { ... }
+  }
+  ```
+- [You can now write open "from" range patterns (`X..`), which will start at `X` and
+  will end at the maximum value of the integer.][83918]
+- [You can now access `union` fields in `const` contexts.][85769]
+- [You can now explicitly import the prelude of different editions
+  through `std::prelude` (e.g. `use std::prelude::rust_2021::*;`).][86294]
+
+Compiler
+--------
+- [Added tier 3\* support for `powerpc-unknown-freebsd`.][87370]
+- [Added tier 3 support for `powerpc64le-unknown-freebsd`.][83572]
+
+\* Refer to Rust's [platform support page][platform-support-doc] for more
+   information on Rust's tiered platform support.
+
+Libraries
+---------
+
+- [Updated std's float parsing to use the Eisel-Lemire algorithm.][86761]
+  These improvements should in general provide faster string parsing of floats,
+  no longer rejects certain valid floating point values, and reduce
```

Comment:
```suggestion
  no longer reject certain valid floating point values, and reduce
```
This should probably say "reject" because "improvements" is plural.